formidable/serializers/forms.py

```diff
@@ -170,3 +172,32 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super(ContextFormSerializer, self).__init__(*args, **kwargs)
         self.fields['fields'].set_context('role', self._context['role'])
+
+
+def get_acccess(accesses, role):
+    for access in accesses:
```

Comment:
maybe missing: a tiny docstring here. the function is self-explanatory, so it might be optional